src/Microsoft.ML.Featurizers/ToStringTransformer.cs

```diff
@@ -0,0 +1,1537 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// This is Auto Generated code used that is being used to unblock other teams.
+// This functionality will be integrated into the existing type conversions over the next several weeks.
+
+using System;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Security;
+using System.Text;
+using Microsoft.ML;
+using Microsoft.ML.CommandLine;
+using Microsoft.ML.Data;
+using Microsoft.ML.EntryPoints;
+using Microsoft.ML.Featurizers;
+using Microsoft.ML.Internal.Utilities;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Transforms;
+using Microsoft.Win32.SafeHandles;
+using static Microsoft.ML.Featurizers.CommonExtensions;
+
+[assembly: LoadableClass(typeof(ToStringTransformer), null, typeof(SignatureLoadModel),
+    ToStringTransformer.UserName, ToStringTransformer.LoaderSignature)]
```

Comment:
Shouldn't `ToStringTransformer` be part of the `ConvertTransform` ([info](https://github.com/dotnet/machinelearning/blob/0a5514b8d1a8a597976aecec6f463e4471b73d6d/docs/code/IDataViewTypeSystem.md#standard-conversions), [info](https://docs.microsoft.com/en-us/dotnet/api/microsoft.ml.conversionsextensionscatalog.converttype?view=ml-dotnet), [code](https://github.com/dotnet/machinelearning/blob/e19369b407e8630cfaa82b07f81f0576c9bbd145/src/Microsoft.ML.Data/Data/Conversion.cs))?

The Convert transform's purpose is to convert the type of columns, for instance, convert a String to a Float, or an Int32 to Int64.

If I recall, the `ConvertTransform` didn't allow conversion to the String type from numeric; I'm unsure the reasoning, but it could be added.


Historically, the Expression transform does make the ToString() conversion easy: `xf=Expression{ col=ColAsString:MyNumericColumn expr={(x) : text(x)} }`, and this is [available in ML.NET](https://github.com/dotnet/machinelearning/blob/354798dc2d9431f5a6a9f27a46b2f50a7f2e3f57/docs/api-reference/expression-estimator.md#functions) (see: "​convert to text").

Reply:
You are correct Justin. This is going in as is for now to unblock the work on ONNX export, and then over the next several weeks I will be converting this, as well as PRs 4594 and 4597, to be in the correct ML.NET transformer, and then removing these classes.

Reply:
Does the Expression transform export to ONNX?

Reply:
There's an issue logged for ONNX for the Expression transform: https://github.com/dotnet/machinelearning/issues/4615

Hopefully the ONNX support is added. The particular AutoML ask was for basic math functions { `log(a)`, `exp(a)`, `a-b` }. With ONNX output for the Expression transform, it would cover those needs. Otherwise the discussion was making independent transforms for each of log/exp/subtract, which would end up with an endless parade of trivial transforms.

Reply:
/cc @davidbrownellWork : My recommendation for implementing the requested `log(a)`, `exp(a)`, and `a-b` transforms is the provide ONNX export for the Expression transform.